src/vcpkg-test/strings.cpp

```diff
@@ -35,6 +35,17 @@ TEST_CASE ("b32 encoding", "[strings]")
     }
 }
 
+TEST_CASE ("percent encoding", "[strings]")
+{
+    std::string ascii(127, 0);
+    std::iota(ascii.begin(), ascii.end(), static_cast<char>(1));
+    REQUIRE(vcpkg::Strings::percent_encode(ascii) ==
+            "%01%02%03%04%05%06%07%08%09%0A%0B%0C%0D%0E%0F%10%11%12%13%14%15%16%17%18%19%1A%1B%1C%1D%1E%1F%20%21%22%23%"
+            "24%25%26%27%28%29%2A%2B%2C-.%2F0123456789%3A%3B%3C%3D%3E%3F%40ABCDEFGHIJKLMNOPQRSTUVWXYZ%5B%5C%5D%5E_%"
+            "60abcdefghijklmnopqrstuvwxyz%7B%7C%7D~%7F");
+    REQUIRE(vcpkg::Strings::percent_encode("👍🌏") == "%F0%9F%91%8D%F0%9F%8C%8F");
```

Comment:
Is there a specific reason for that (like compilers or IDEs do not support utf8)?

Reply:
Yeah, back when I was on the STL team we got bit by this (people on non-English Windows getting compiler spew because the file was interpreted as some other code page).

It probably doesn't matter here but ...